proto/bitsong/cadence/v1/tx.proto

```
@@ -0,0 +1,99 @@
+syntax = "proto3";
+package bitsong.cadence.v1;
+
+option go_package = "github.com/bitsongofficial/go-bitsong/x/cadence/types";
+
+import "google/api/annotations.proto";
```

Buf (1.47.2)

import "google/api/annotations.proto": file does not exist

(COMPILE)